api/proto/config/devmodel.proto

```diff
@@ -64,15 +70,20 @@ message SystemAdapter {
         // deprecate: have a separate device policy object in the API
         bool uplink = 3;
 
+        // XXX Note that this is the static information from the model.
```

Comment:
It doesn't make sense to change this in this direction at this point in time.
I do think we need to introduce a separate policy object e2e (from UI via controller to EVE) which can be used tp specify what different system adapters can be used for.